AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -671,3 +671,68 @@ def run_fbx_test(self, workspace, ap_setup_fixture, asset_processor,
+            4. Modify the FBX file
+            5. Run asset processor
+            6. Validate that Asset Processor generates the expected output
+        """
+
+        # Copying test assets to project folder
+        asset_processor.prepare_test_environment(ap_setup_fixture["tests_dir"], "ModifiedFBXFile_ConsistentProductOutput")
+        # Run AP against the FBX file and the .assetinfo file
+        result, _ = asset_processor.batch_process(extra_params=["--debugOutput",
+                                                    "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
+        assert result, "Asset Processor Failed"
+
+        # Set path to expected dbgsg output, copied from test folder
+        scene_debug_expected = os.path.join(asset_processor.project_test_source_folder(), "SceneDebug", "modifiedfbxfile.dbgsg")
+        assert os.path.exists(scene_debug_expected), "Expected scene file missing in SceneDebug/modifiedfbxfile.dbgsg - Check test assets"
+
+        # Set path to actual dbgsg output, obtained when running AP
+        scene_debug_actual = os.path.join(asset_processor.temp_project_cache(asset_platform=ASSET_PROCESSOR_PLATFORM_MAP[workspace.asset_processor_platform]), "ModifiedFBXFile_ConsistentProductOutput","modifiedfbxfile.dbgsg")
+        assert os.path.exists(scene_debug_actual)
+
+        # Compare the dbgsg files to ensure expected outputs
+        self.compare_scene_debug_file(asset_processor, scene_debug_expected, scene_debug_actual)
+
+        # Run again for the .dbgsg.xml files
+        self.compare_scene_debug_file(asset_processor, scene_debug_expected + ".xml", scene_debug_actual + ".xml")
+
+        # Remove the files to be replaced from the source test asset folder
+        filestoremove = [
+            os.path.join(asset_processor.project_test_source_folder(),"ModifiedFBXFile.fbx"),
+            scene_debug_expected,
+            scene_debug_expected + ".xml"
+        ]
+        for file in filestoremove:
+            os.remove(file)
+            assert not os.path.exists(file), f"File failed to be removed: {file}"
+
+        # Add the replacement FBX and expected dbgsg files into the test project
+        source = os.path.join(ap_setup_fixture["tests_dir"], "Assets",
+                              "Override_ModifiedFBXFile_ConsistentProductOutput")
+        destination = asset_processor.project_test_source_folder()
+        # Note: Replace distutils for shutil.copytree("src", "dst", dirs_exist_ok=True) once updated to Python 3.8
+        distutils.dir_util.copy_tree(source, destination)
+        assert os.path.exists(scene_debug_expected), \
+            "Expected scene file missing in SceneDebug/modifiedfbxfile.dbgsg - Check test assets"
+
+        # Run AP again to regenerate the .dbgsg files
+        asset_processor.batch_process(extra_params=["--debugOutput",
+                                                    "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
```

Comment:
maybe "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\"" should be a const variable so it's not copy & pasted so often?

skipAtomoutputParam = "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""